client/signup/steps/site-or-domain/index.jsx

```diff
@@ -2,13 +2,16 @@ import { localize } from 'i18n-calypso';
 import { get, isEmpty } from 'lodash';
 import { Component } from 'react';
 import { connect } from 'react-redux';
+import HeaderImage from 'calypso/assets/images/domains/domain.svg';
 import QueryProductsList from 'calypso/components/data/query-products-list';
 import { domainRegistration } from 'calypso/lib/cart-values/cart-items';
 import { getDomainProductSlug } from 'calypso/lib/domains';
 import StepWrapper from 'calypso/signup/step-wrapper';
 import { isUserLoggedIn } from 'calypso/state/current-user/selectors';
 import { getAvailableProductsList } from 'calypso/state/products-list/selectors';
 import { submitSignupStep } from 'calypso/state/signup/progress/actions';
+import { globe, newSite, site } from '../../icons';
```

Comment:
As said in the other comment, I believe these could be replaced by `import { globe, addCard, layout } from '@wordpress/icons'`. I experimented doing that:

<img width="567" alt="Screen Shot 2022-01-24 at 19 59 26" src="https://user-images.githubusercontent.com/5324818/150879209-d6feaecf-dc58-4558-8022-05b3d7cb10b5.png">

I think only the icons' color would need some tweaking 🙂

Reply:
I found these by going through the [@wordpress/icons](https://github.com/WordPress/gutenberg/tree/trunk/packages/icons/src/library) source, which wasn't ideal 🤔

I tried to find someplace where we can find icons more easily and after some MGS'ing, found this very helpful link: https://wordpress.github.io/gutenberg/?path=/story/icons-icon--library 😄